pubsub/gochannel/pubsub.go

```diff
@@ -11,6 +11,10 @@ import (
 	"github.com/ThreeDotsLabs/watermill/message"
 )
 
+// NoSubscribersFallbackTopic is the fallback topic messages without any subscribers will be sent to.
+// This is used if the `EnableFallback` configuration option is enabled.
+const NoSubscribersFallbackTopic = "*"
```

Comment:
what do you think about adding configuration for the topic name instead? so we could have two options in `Config`

`EnableFallback`
`NoSubscribersFallbackTopic`

what do you think?

Reply:
I added both `EnableNoSubscribersFallback` (as per your review comment below) and `NoSubscribersFallbackTopic` to the configuration. I did want to leave the default topic (`*`) as a const in case the user doesn't set a custom fallback topic. I also added another test case for this.